packages/playwright-test/ThirdPartyNotices.txt

```diff
@@ -6,39 +6,39 @@ This project incorporates components from the projects listed below. The origina
 
 -	@ampproject/remapping@2.2.0 (https://github.com/ampproject/remapping)
 -	@babel/code-frame@7.18.6 (https://github.com/babel/babel)
--	@babel/compat-data@7.20.1 (https://github.com/babel/babel)
--	@babel/core@7.20.2 (https://github.com/babel/babel)
--	@babel/generator@7.20.4 (https://github.com/babel/babel)
+-	@babel/compat-data@7.20.14 (https://github.com/babel/babel)
```

Comment:
There should be no changes other than the json5.

Reply:
In playwright-core/packages/bundles, json5 is not present in package.json but it is coming as dependency of @babel/core. So on updating @babel/core only we can get json5 updated.